doc/rfcs/login_block_sliding_window.md

```diff
@@ -0,0 +1,66 @@
+# Login Sliding-Window Blocks
+
+## Overview and motivation
+`ms-users` service supports some IP and username blocks. The Current version uses block on TTL'ed counter in the Redis database,
+but this brings some strong limits for users. Service monitors count of login attempts from IP address and IP + Username,
```

Comment:
Service monitors count login attempts by two factors: IP address and IP + Username. If one of the counters exceeds specified config parameter value, all requests identified by this factor are blocked until TTL is expired.